src/Fields/field.jl

```diff
@@ -157,7 +157,7 @@ function Field(loc::Tuple,
 end
     
 Field(z::ZeroField; kw...) = z
-Field(f::Field) = f # hmm...
+Field(f::Field; indices=f.indices) = view(f, indices...) # hmm...
```

Comment:
:+1: this is probably all that was needed to fix the bug

Reply:
```suggestion
Field(f::Field; indices=f.indices) = view(f, indices...)
```

Reply:
There's still a few uncertain things here, like whether we should respect / include other keyword arguments (boundary conditions, data, etc...)

Reply:
Then can we include a more descriptive comment please?

Also, doesn't removing the line `Field(f::Field) = f` mean that `Field(u)` won't work anymore? Maybe we wanna keep that line?

Reply:
`Field(u)` works; the kwarg is default `u.indices` (eg the same indices)